services/wallet/router/fees.go

```diff
@@ -141,9 +137,9 @@ func (f *FeeManager) SuggestedFees(ctx context.Context, chainID uint64) (*Sugges
 			BaseFee:              big.NewInt(0),
 			MaxPriorityFeePerGas: big.NewInt(0),
 			MaxFeesLevels: &MaxFeesLevels{
-				Low:    (*hexutil.Big)(big.NewInt(0)),
-				Medium: (*hexutil.Big)(big.NewInt(0)),
-				High:   (*hexutil.Big)(big.NewInt(0)),
+				Low:    (*hexutil.Big)(gasPrice),
+				Medium: (*hexutil.Big)(gasPrice),
+				High:   (*hexutil.Big)(gasPrice),
```

Comment:
that used to be there to detect non eip1559 chain but there is no such chain anymore right?

Reply:
Everything is the same, I added `func (m *MaxFeesLevels) feeFor(mode GasFeeMode) *big.Int`, that way we're getting appropriate MaxFee based on the passed `GasFeeMode`, then I updated `func (s *SuggestedFees) feeFor(mode GasFeeMode) *big.Int` which previously had `if !s.EIP1559Enabled` condition, under which it was returning `GasPrice`, but now in the case of `EIP1559Enabled` we're returning default `MaxFee` of `MaxFeesLevels ` which is set to `GasPrice`, so basically the effect is the same, just written differently. We can revert this part and nothing will be changed.